### DISCUSSION TITLE
Issue with hbl CVA plot and axes/setColorRange

### DISCUSSION THREAD

**Discussion by ginnyyb** [mtex-toolbox/mtex#1275] [Ask Anything]
I am currently trying to produce CVA plots for the bulk vorticity as well as individual phases and am having trouble getting hornblende to plot in one sample (in specimen AND geographic reference frame). I was able to successfully produce a CVA plot for hbl in another sample as well as all other phases + bulk vorticity within this same sample, so I'm wondering if there is an issue with file size for this particular one. I am still relatively new to MTEX and do not recognize these errors or how to correct this issue if anybody has had experience with these error messages (seen in attached [screenshot)[](url)](ur
![Screen Shot 2021-11-09 at 12 12 20 PM](https://user-images.githubusercontent.com/90217800/140997904-7ab29bb3-75c4-485e-9410-af3bc7d23253.png)
l)

**Comment by zmichels** [comment]
Hi Ginny,

Try removing or commenting-out the line that starts with “setColorRange(…”
The error is indicating that something about the color limits specified in the code aren’t matching with the data. My *guess* is that where the code specifies a minimum of MUD = 1… perhaps the data being plotted have a maximum MUD that is less-than 1… such that Matlab doesn’t know what to do and throws an error. That may not be the specific problem… but the error is stemming from that part of the plotting routine… so I *think* it should get passed that part if you remove it (or comment t out), or if you change the 1 to a 0.

Let me know if the works!
Zach

**Reply by ginnyyb** [reply]
Hi Zach,

Thank you for such quick feedback! Removing the setColorRange certainly helped eliminate those errors, but the contour CVA data is still not plotting on the pole figures. I couldn't find anywhere on either of my scripts (import script + CVA script) that specified an MUD value, so I added "MUD = 0;" and it didn't seem to make a difference. I am still getting the warning "Contour not rendered for non-finite ZData" and am wondering if this may be indicative of the issue?
![Screen Shot 2021-11-09 at 1 06 26 PM](https://user-images.githubusercontent.com/90217800/141005179-48fad5d5-433b-4531-a033-dcc661451f04.png)
?

Cheers,
Ginny

**Comment by zmichels** [comment]
Hi Ginny,

Can you link/send me the dataset directly ***@***.***), rather than through the forum? I’ll try it myself tonight. 

The “MUD” is just the colorrange values for the density/contour plot that were put in that line, but as you mention you get the error for ZData even after removing those lines. 

I admit I’ve never had this trouble or this exact error, but I’m guessing it’s something about the range of the data being sent to the plotting command.

We’ll get it sorted tho.

Cheers,
Zach

**Reply by ginnyyb** [reply]
Hi Zach,

Sorry to get back to you so late but I actually found my mistake - there is no hornblende in that sample! I had mixed up that sample # with another sample. Glad it was an easy fix, but unfortunate that I didn't catch this oversight before asking for help.

Thanks,
Ginny

**Comment by zmichels** [comment]
Hi Ginny,

No worries, I’m just glad to hear you got it sorted!

Cheers,
Zach
